### ISSUE TITLE
Not working in Windows 7 with Firefox V75.0

### ISSUE THREAD

**Issue by Kwsidereal** [dauphine-dev/drop-feeds#121]
Note: Don't hesitate to add screenshots 

#### System: 
 * Operating System:  Windows 7 64-bit
 * Firefox version: 75.0
 * Drop Feeds version : 1.0.3a

#### Steps to reproduce:
Was installed in Firefox V-73.0, I recall it worked there a few months ago.
Updated Firefox to V-75.0, Drop Feeds extension loaded and displayed but could not subscribe to any RSS, like BBC News.
Deleted Drop Feeds and re-installed, but still not able to subscribe. When I click on the "Subscribe" button nothing happens, no dialog box to enter RSS URL.

#### What should happen:
When "Subscribe" button is clicked nothing happens.
"Discover feeds" button causes dialog box but no feed discovered. Also, "Add feed" button does nothing.

#### Addition information:
Drop feeds works in Windows 10 but not 7.

**Comment by dauphine-dev** [comment]
Hi @Kwsidereal,
I have just tested Drop Feeds 1.0.3a in Firefox 75 on Windows 7 64-bit,
I can subscribe to any RSS feeds (including BBC News).
Then please can you open the debug console, 
see here: [How to display the debug console](https://github.com/dauphine-dev/drop-feeds/wiki/How-to-display-the-debug-console-%3F)
Clean the console, try to subscribe to a RSS feed, and put the console output.

And can you upload a full export of your Drop Feeds too, see here: [How to do a full export](https://github.com/dauphine-dev/drop-feeds/wiki/How-to-do-a-full-export)

**Comment by Kwsidereal** [comment]
Hi dauphine-dev,
Update to my previous bug report:
Sorry, but my initial impression about not being able to subscribe to RSS feeds was wrong.
I again tried Drop Feeds in a fresh install of Linux with Firefox 75.0, and at first it looked like it had the same problem, as there was no 7th button at the top of the sidebar for Options/Subscribe by URL, but I was assuming that the sidebar opened in full width - turns out it was not, and so concealed the 7th button for Options. Fixed by resizing the sidebar a bit wider.

This did not occur to me because the first time I installed Drop Feeds several months ago, the sidebar was wide enough to show all 7 buttons. Well, I guess not thinking of simple solutions is what happens when you get old.

Amitiés sincères,
Kwsidereal.
